crates/oxc_transformer/src/plugins/styled_components.rs

```diff
@@ -894,8 +892,8 @@ fn minify_template_literal<'a>(lit: &mut TemplateLiteral<'a>, ast: AstBuilder<'a
     let mut output = Vec::new();
 
     // TODO: What about `cooked`? Shouldn't we alter that too?
-    let mut quasi_index = 0;
-    while quasi_index < quasis.len() {
+    let quasis = &mut lit.quasis[..];
+    for quasi_index in 0..quasis.len() {
         let mut bytes = quasis[quasi_index].value.raw.as_str().as_bytes();
```

Comment:
Can we use `iter().enumerate()` here?

Reply:
Unfortunately not. Borrow-checker will not allow. The iterator would borrow `quasis`, so then you can't mutate other elements of `quasis` in the loop e.g. `quasis[quasi_index - 1].span = REMOVE_SENTINEL;`.

Reply:
We could go unsafe code on it and get around this. But that's an optimization too far, even for me!

Reply:
Ok, I forgot that part